test/Constraints/implicit_double_cgfloat_conversion.swift

```diff
@@ -216,3 +216,49 @@ func test_multi_argument_conversion_with_optional(d: Double, cgf: CGFloat) {
 
   test(cgf, d) // Ok (CGFloat -> Double and Double? -> CGFloat?)
 }
+
+extension CGFloat: Hashable {
+  public func hash(into hasher: inout Hasher) { fatalError() }
+}
+
+func test_array_literal_as_call_argument() {
+  enum E {
+  case test_arr([CGFloat])
+  case test_dict_key([CGFloat: String])
+  case test_dict_value([String: CGFloat])
+  case test_arr_nested([String: [[CGFloat]: String]])
+  case test_dict_nested([String: [String: CGFloat]])
+  }
```

Comment:
Spotted unindented cases. Disregard if this is a stylistic preference.

Reply:
Fixed!